app/helpers/view_helper.rb

```diff
@@ -37,12 +37,12 @@ def enrichment_error_url(provider_code:, course:, field:)
     base = "/organisations/#{provider_code}/#{course.recruitment_cycle_year}/courses/#{course.course_code}"
 
     {
-      'about_course' => base + '/about#about_course_wrapper',
-      'how_school_placements_work' => base + '/about#how_school_placements_work_wrapper',
-      'fee_uk_eu' => base + '/fees#fee_uk_eu_wrapper',
-      'course_length' => base + (course.has_fees? ? '/fees' : '/salary') + '#course_length_wrapper',
-      'salary_details' => base + '/salary#salary_details_wrapper',
-      'qualifications' => base + '/requirements#required_qualifications_wrapper'
+      'about_course' => base + '/about?display_errors=true#about_course_wrapper',
+      'how_school_placements_work' => base + '/about?display_errors=true#how_school_placements_work_wrapper',
+      'fee_uk_eu' => base + '/fees?display_errors=true#fee_uk_eu_wrapper',
+      'course_length' => base + (course.has_fees? ? '/fees' : '/salary') + '?display_errors=true#course_length_wrapper',
+      'salary_details' => base + '/salary?display_errors=true#salary_details_wrapper',
+      'qualifications' => base + '/requirements?display_errors=true#required_qualifications_wrapper'
```

Comment:
When would `display_errors` ever be missing/false? And would `@course.errors` ever be populated in that case?

Reply:
It's not there when you visit `/fees` directly

Reply:
I can't quite think of how `@course.errors` would have any errors in that case, though.

Reply:
@dankmitchell Do you know if we need this added bit machinery `display_errors`? I'd be keen to strip out anything we don't need.

Reply:
It's the only way we found to trigger re-validation in the controller for each page...

Reply:
I see now. `display_errors` is used to avoid posting to the backend needlessly. This makes complete sense. :shipit: